script/texdoclib-view.tlu

```diff
@@ -211,31 +216,78 @@ local function print_menu(name, doclist, showall)
     end
 end
 
+local function search_online(name, doclist)
+    if not texdoc.config.get_value('interact_switch') or
+       texdoc.config.get_value('machine_switch')
+    then
+        view_doc(doclist[1])
+        return
+    end
+
+    local function write(msg)
+        io.stderr:write(msg .. '') -- cast to string
+    end
+    -- Fuzzy heuristic to see if the user has _any_ local documentation
+    local texdoc_docs = texdoc.search.get_doclist("texdoc")
+    local docs_installed = texdoc[1] and texdoc[1].basename == "texdoc.pdf"
+
+    if docs_installed then
+        write(C.badmatch_msg:gsub(C.badmatch_msg_ph, name))
+
+        for i=1,3 do
+            local doc = (doclist[i] and doclist[i].normname) or '<empty>'
+            write('    ' .. i .. ' ' .. doc .. '\n')
+        end
+        write('\n')
+    else
+        write(C.nolocaldocs_msg)
+    end
+
+    local url = C.online_baseurl:gsub(C.online_baseurl_ph, name)
+    write(C.online_msg:gsub(C.online_msg_url, url)
+                      :gsub(C.online_msg_ph, name))
+
+    if docs_installed then
+        write(C.badmatch_prompt)
+    else
+        write(C.nolocaldocs_prompt)
+    end
+
+    local ans = io.read('*line')
+
+    if ans == 'y' or ans == 'Y' then
```

Comment:
Let us just try our best not to step on a bug (just in case.)

https://github.com/TeX-Live/texdoc/blob/a27708f201ce4216edf7308d9a92ecdb005b294f/script/texdoclib-view.tlu#L176-L178

Reply:
I'll get that changed. Thanks!